### PR TITLE
Add tutorial for visualizing volume data with Python, update docs

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Comment by nilsvu** [comment]
@yoonso0-0 mind taking a quick look?

**Comment by nilsvu** [comment]
@yoonso0-0 and/or @knelli2 please take a look at this

**Comment by yoonso0-0** [comment]
I'll try to look at this before the end of this week

File: docs/Tutorials/VisualizationWithPython.ipynb
Comment (knelli2) [inline docs/Tutorials/VisualizationWithPython.ipynb:2]: So we already have a "Visualizing volume data" [tutorial](https://spectre-code.org/tutorial_visualization.html). This feels like a very natural extension of that tutorial, having two sections

- Visualizing with paraview GUI
- Visualizing with spectre python

File: docs/Tutorials/VisualizationWithPython.ipynb
Comment (knelli2) [inline docs/Tutorials/VisualizationWithPython.ipynb:13]: Maybe add a link to the `python.md` tutorial you just edited since that has detailed instructions on how to load the spectre module

File: docs/Tutorials/VisualizationWithPython.ipynb
Comment (knelli2) [inline docs/Tutorials/VisualizationWithPython.ipynb:271]: Remove?

**Review by knelli2 — CHANGES_REQUESTED** [review]

File: docs/Tutorials/VisualizationWithPython.ipynb
Comment (nilsvu) [reply docs/Tutorials/VisualizationWithPython.ipynb:2]: I added a section there with a link to the notebook (can't merge the tutorials because this is a notebook).

**Comment by nilsvu** [comment]
Rebased and updated this @knelli2.

@yoonso0-0 mind taking a look as well?

File: docs/Tutorials/Cli.md
Comment (yoonso0-0) [inline docs/Tutorials/Cli.md:31]: So I tried this on develop, but this didn't make any change on my `zsh`. It actually is effective after sourcing `shell-completion.bash`.

File: docs/Tutorials/VisualizationWithPython.ipynb
Comment (yoonso0-0) [inline docs/Tutorials/VisualizationWithPython.ipynb:63]: Please add one-line comment like
```
# Change filename to your sim data
```

File: docs/Tutorials/VisualizationWithPython.ipynb
Comment (yoonso0-0) [inline docs/Tutorials/VisualizationWithPython.ipynb:70]: Can we also add a cell for printing out a list of available variables?

**Review by yoonso0-0 — CHANGES_REQUESTED** [review]

File: docs/Tutorials/Cli.md
Comment (nilsvu) [reply docs/Tutorials/Cli.md:31]: It works for me in ZSH with the instructions here when I have the `bin/` directory in the `PATH`, so I can do just `spectre <TAB>`. It doesn't work for me when I do `./bin/spectre <TAB>`. Seems fine I think. Edit: Actually `./bin/spectre` seems to work too.

**Comment by nilsvu** [comment]
Updated with your suggestions @yoonso0-0. Thanks for reviewing!

**Comment by nilsvu** [comment]
Please look at again @knelli2. Seems ready to merge.

**Comment by knelli2** [comment]
@nilsvu will update with tutorial from SXS-Con

**Comment by nilsvu** [comment]
@knelli2 you can actually just merge this PR. Then I can update the tutorial with the additions from sxscon in a next PR.

**Comment by knelli2** [comment]
@nilsvu Can you give this a quick rebase? I'll merge after CI passes

**Review by knelli2 — APPROVED** [review]

**Pull request merged**
